federatedscope/core/fed_runner.py

```diff
@@ -102,6 +115,8 @@ def _setup_for_standalone(self):
         for client_id in range(1, self.cfg.federate.client_num + 1):
             self.client[client_id] = self._setup_client(
                 client_id=client_id, client_model=self._shared_client_model)
+            if client_id in unseen_clients_id:
+                self.client[client_id].is_unseen_client = True
```

Comment:
Like above

Reply:
moved it into the setup_server() as sub action of init